app/routes.js

```diff
@@ -63,8 +63,13 @@ export const routes = (authProvider) => {
     res.render('pages/organisation/template.njk', addContext({ context, user: req.user }));
   }));
 
-  router.post('/organisations/:organisationId', authProvider.authorise(), withCatch(authProvider, async (req) => {
-    logger.info('POST /organisations/:organisationId', req.body);
+  router.post('/organisations/:organisationId', authProvider.authorise(), withCatch(authProvider, async (req, res) => {
+    const { organisationId } = req.params;
+    const accessToken = extractAccessToken({ req, tokenType: 'access' });
+    await putUpdateOrganisation({ organisationId, body: req.body, accessToken });
+    logger.info(`redirecting to edit organisation: ${organisationId} confirmation page`);
+    // TODO: change to /organisations/${organisationId}/edit/confirmation when confirmation is done
```

Comment:
ooops or are you going to rename the route do when the confirmation page is done?